ingest/core/query_test.py

```
@@ -0,0 +1,209 @@
+        print_search_results(results, args.query)
+    elif args.sample_queries:
+        # Sample queries
+        run_sample_queries(vector_store)
+    elif args.interactive:
+        # Interactive mode
+        interactive_query_mode(vector_store)
+    else:
+        # Default: show stats and run interactive mode
+        stats = vector_store.get_index_stats()
+        print("📊 Index Statistics:")
+        for key, value in stats.items():
```

Ruff (0.13.1)

Loop control variable `key` not used within loop body

Rename unused `key` to `_key`

(B007)